### DISCUSSION TITLE
Dropdown empty space error

### DISCUSSION THREAD

**Discussion by andyargg** [flet-dev/flet#2463] [Q&A] [answered]
### Question

How can i validate an empty space in a dropdown/textfield in flet?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ingdesarr1** [comment, marked as answer]
You can use the value property, and validate if value is None or equal to ''

**Reply by andyargg** [reply]
def validate_form(self, e):
        for dropdown in self.dropdowns:
            #USAR REGEX ACA 
            if (dropdown.value) == None or dropdown.value == "":
                # print("error")
                dropdown.error_text="Selecciona una opcion"
            else:
                dropdown.error_text=""
                
                
                
                
                
                

I did it like this and worked
